sw/device/tests/sim_dv/BUILD

```diff
@@ -35,6 +35,9 @@ opentitan_functest(
     # loading the default test ROM, or any other ROM that may be specified via
     # Verilator or CW310 params).
     test_in_rom = True,
+    verilator = verilator_params(
+        tags = ["failing_verilator"],
```

Comment:
Can you associate this with a github issue?

Reply:
Done - referenced #12486